test/suites/drift/suite_test.go

```diff
@@ -288,34 +281,29 @@ var _ = Describe("Drift", func() {
 
 			// Make 5 pods all with different deployments and different test partitions, so that each pod can be put
 			// on a separate node.
-			selector = labels.SelectorFromSet(appLabels)
-			numPods = 5
-			deployments := make([]*appsv1.Deployment, numPods)
-			for i := range lo.Range(numPods) {
-				deployments[i] = coretest.Deployment(coretest.DeploymentOptions{
-					Replicas: 1,
-					PodOptions: coretest.PodOptions{
-						ObjectMeta: metav1.ObjectMeta{
-							Labels: appLabels,
-						},
-						NodeSelector: map[string]string{"test-partition": fmt.Sprintf("%d", i)},
-						// Each xlarge has 4 cpu, so each node should fit no more than 1 pod.
-						ResourceRequirements: v1.ResourceRequirements{
-							Requests: v1.ResourceList{
-								v1.ResourceCPU: resource.MustParse("3"),
-							},
-						},
+			deployment := coretest.Deployment(coretest.DeploymentOptions{
```

Comment:
This would make 1 deployment with 5 pods, don't we want 5 deployment with 1 pods?

Reply:
It shouldn't make a difference, the 5 deployments allowed us to create 5 different virtual domains to ensure the pods scheduled onto different nodes. Seems like the CPU request was also there for the same purpose. Anti-affinity should have the same result.